ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
-<!ENTITY      tab.signonsstored.label         "Сохранённые пароли">
-<!ENTITY      tab.signonsnotstored.label      "Пароли никогда не будут сохраняться">
-<!ENTITY      tab.nopreview.label             "Предварительно введённые в форму данные никогда не будут сохраняться">
-<!ENTITY      tab.nocapture.label             "Введённые в форму данные никогда не будут сохраняться">
+<!ENTITY      tab.signonsstored.label         "Сохраненные пароли">
+<!ENTITY      tab.signonsnotstored.label      "Никогда не сохранять пароли">
+<!ENTITY      tab.nopreview.label             "Отключить предварительный просмотр форм">
+<!ENTITY      tab.nocapture.label             "Никогда не сохранять формы">
 
-<!ENTITY      spiel.signonsstored.label       "Менеджер паролей будет сохранять информацию о логинах для следующих сайтов:">
-<!ENTITY      spiel.signonsnotstored.label    "Менеджер паролей никогда не будет сохранять информацию о логинах для следующих сайтов:">
-<!ENTITY      spiel.nopreview.label           "Менеджер форм никогда не будет сохранять предварительно введённые в формы данные для следующих сайтов до их заполнения Вами:">
```

Comment:
I asked Sherman about this line and replied: "You can skip that one."
So, we can leave it as it is.